UI/backend/db_utils.py

```diff
@@ -0,0 +1,114 @@
+import psycopg2
+import traceback
+
+def connect_to_db():
+    try:
+        connection = psycopg2.connect(
+            host="localhost",
```

Comment:
you have a `config.json` where db connection info is written. Instead of using hardcoded values, import that config and use it

Reply:
done